### DISCUSSION TITLE
Typescript parser sync to async migration cannot find files

### DISCUSSION THREAD

**Discussion by Jahnavi-Tirunagari** [webdriverio/codemod#111] [Migration Issue] [answered]
I am trying to follow the steps for sync to async migration with relative path in a windows system and I always get the following output
`No files selected, nothing to do.
All done.
Results:
0 errors
0 unmodified
0 skipped
0 ok
Time elapsed: 0.001seconds`
Example command: npx jscodeshift -t ./node_modules/@wdio/codemod/async --parser=tsx .\pages\fw\unified_nav\*.ts

**Comment by christian-bromann** [comment]
Can you try with:

```sh
npx jscodeshift -t ./node_modules/@wdio/codemod/async --parser=tsx ./pages/fw/unified_nav*.ts
```

I've never tested this on Windows so chances are that using Window style paths is broken.

**Reply by Jahnavi-Tirunagari** [reply, marked as answer]
After a couple tries, this is what worked for me
`npx jscodeshift -t ./node_modules/@wdio/codemod/async --extensions=tsx,ts --parser=ts .\pages\fw\unified_nav\`

**Comment by bajipogadadanda** [comment]
will it work for java script(js )also, if we change the  extension and parser?

**Reply by christian-bromann** [reply]
This codemod should work for `.js` files as well.
